src/components/Pages/PeoplePage.1.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { ErrorMessages } from '../../types/ErrorMessages';
```

Comment:
Ensure that the `ErrorMessages` type is correctly defined and imported from the correct path. Verify that `../../types/ErrorMessages` is the correct path and that it exports the necessary constants or types.